src/sentry/tasks/codeowners/update_code_owners_schema.py

```diff
@@ -23,11 +32,11 @@ def update_code_owners_schema(organization, integration=None, projects=None, **k
             projects = [load_model_from_db(Project, project) for project in projects]
             code_owners = ProjectCodeOwners.objects.filter(project__in=projects)
 
-        if integration:
-            integration = load_model_from_db(Integration, integration, allow_cache=False)
+        integration_id = _unpack_integration_id(integration)
+        if integration_id is not None:
             code_mapping_ids = RepositoryProjectPathConfig.objects.filter(
                 organization_integration__organization_id=organization.id,
-                organization_integration__integration_id=integration.id,
+                organization_integration__integration_id=integration_id,
             ).values_list("id", flat=True)
```

Comment:
We may need to rework this further to avoid queries. The code on master has changed from when you branched off, so you likely have conflicts here.

Reply:
Yep, will merge.